bash_scripts/setup.bash

```diff
@@ -20,16 +20,20 @@ if [ ! -d $test_scratch_dir_atm ]; then
   mkdir $test_scratch_dir_atm
 fi
 
-if [ ! -d $ref_scratch_dir ] && [ $ref_case != "obs" ]; then
+#if [ ! -d $ref_scratch_dir ] && [ $ref_case != "obs" ]; then
+#  mkdir $ref_scratch_dir
+#fi
+
+if [ ! -d $ref_scratch_dir ]; then
   mkdir $ref_scratch_dir
 fi
 
-if [ "$ref_case" != "obs" ]; then
+#if [ "$ref_case" != "obs" ]; then
   ref_scratch_dir_atm=$ref_scratch_dir/atm
   if [ ! -d $ref_scratch_dir_atm ]; then
     mkdir $ref_scratch_dir_atm
   fi
-fi
+#fi
```

Comment:
we should remove the commented out lines at some point before merging.

Reply:
yes, @milenaveneziani. I will do that. Regarding the scratch directory, I noticed earlier that there has been a slight modification in the path to the scratch space. I remember it being something like $projdir/$casename.scratch. But, now its $projdir/$plot_dir_name.scratch. $plots_dir_name is specific to the prescribed years. So, whenever we run the diags with new years, it would create a new directory and would now know that some files may already exist. Right? Should we change it to $projdir/$casename.scratch?